Citizens/templates.yml

```diff
@@ -1246,6 +1246,38 @@ ironmanmkxv:
     uuid: 9b3e4ff2-136d-481c-8aa7-f858db93265b
     traitnames: spawned,type,inventory,owner,gravity,location
     name: sneaky
+    terminator:
```

Comment:
incorrect spacing for terminator node
also should be renamed since there is more than 1 terminator model